docs/extend/extend-cli/cli-creating-plug-in-lifecycle-actions.md

```diff
@@ -1,40 +1,42 @@
 # Creating plug-in lifecycle actions
 
-As a developer, you may want your plug-in to perform certain tasks immediately after install and just before uninstall.
+As a developer, you might want your plug-in to perform certain tasks immediately after install and just before uninstall.
 
 Many different types of tasks can make up these *plug-in lifecycle actions*, including the following examples:
 
 - Post-install actions:
-    - A sanity check 
+    - An initial check
     - Additional setup
-    - Adding the plug-in as an override of Zowe CLI Credential Manager
+    - Adding the plug-in as an override of Zowe CLI's built-in credential manager
 
 - Pre-uninstall actions:
     - Revert specialized setup
-    - Removing the plug-in as an override of Zowe CLI Credential Manager
+    - Removing the plug-in as an override of Zowe CLI's built-in credential manager
 
 Creating and using lifecycle actions is optional, but they can be useful tools. Lifecycle actions can automate a manual process intended for the plug-in user to carry out. They can also avoid the need to create commands with uses limited to post-install and pre-uninstall tasks.
 
-**Note:** When creating a plug-in to override Zowe CLI Credential Manager, it is necessary to implement a post-install action to configure your plug-in as the credential manager.
+:::note
+When creating a plug-in to override Zowe CLI's built-in credential manager, it is necessary to implement a post-install action to configure your plug-in as the credential manager.
+:::
 
 ## Implenting lifeycyle actions
 
-Add the `pluginLifeCycle` property to your plug-in definition file and include a plug-in class to implement lifecycle functions.
-
-Follow these steps:
+To add the `pluginLifeCycle` property to your plug-in definition file and include a plug-in class to implement lifecycle functions:
 
 1. Navigate to the plug-in definition file.
 
     This file is the value for the `configurationModule` property in the plug-in `package.json` file.
 
-    See this [`IImperativeConfig.ts` file](https://github.com/zowe/imperative/blob/master/packages/imperative/src/doc/IImperativeConfig.ts) to view an example of the detailed format used in the plug-in definition file.
+    See this [`IImperativeConfig.ts` file](https://github.com/zowe/imperative/blob/master/packages/imperative/src/doc/IImperativeConfig.ts) to view an example of the detailed format used in the plug-in definition file. **[this repo is archived. what else can we link to here?]**
```

Comment:
Change to "To view an example of the detailed format used in the plug-in definition file, see ..."